examples/deferred_processing/automated_deferred_processing/Makefile

```
@@ -0,0 +1,102 @@
+
+# The executable paths below are set to generic values.
+# Modify them for your system by setting environment variables, using a command like this
+# to both fetch the version of pachyderm you want to use and execute it.
+# "../etc/fetch_release_pachctl.py 1.10.0 ; env PACHCTL=${GOPATH}/bin/pachctl make -e opencv"
+
+SHELL := /bin/bash
+PACHCTL := pachctl
+KUBECTL := kubectl
+DOCKER := docker
+
+DEFERRED_PROCESSING_PIPELINES = ../deferred_processing_plus_transactions/edges_dp.json \
+				../deferred_processing_plus_transactions/montage_dp.json 
+
+DEFERRED_PROCESSING_1_FILES = ../deferred_processing_plus_transactions/images.txt \
+				../deferred_processing_plus_transactions/images2.txt
+
+DEFERRED_PROCESSING_2_FILES = ../deferred_processing_plus_transactions/images3.txt
+
+
+CONTAINER_VERSION := 1.11.0
+DOCKER_ACCOUNT := pachyderm
+CONTAINER_NAME := pachctl
+CONTAINER_TAG := $(DOCKER_ACCOUNT)/$(CONTAINER_NAME):$(CONTAINER_VERSION)
```

```suggestion
CONTAINER_IMAGE := pachyderm/pachctl:1.11.0
```

1) This is the only necessary variable here, removing the others simplifies things a bit
2) This would be more appropriately named `CONTAINER_IMAGE`